release/benchmarks/many_nodes.yaml

```diff
@@ -14,11 +14,13 @@ head_node_type:
 
 worker_node_types:
     - name: small_worker
-      instance_type: m5.2xlarge
+      instance_type: m6i.2xlarge
       min_workers: 249
       max_workers: 249
       use_spot: false
       resources:
         custom_resources:
           node: 1
 
+flags:
+  allow-cross-zone-autoscaling: true
```

Comment:
We should always run in one AZ to make the perf predictable.

Reply:
removed flag, still works https://buildkite.com/ray-project/release/builds/27759#0193b240-9210-448f-9197-1f7954ffbf7a, was able to acquire ~3 minutes